HIRS_Utils/src/main/java/hirs/data/persist/certificate/attributes/ComponentIdentifier.java

```diff
@@ -99,9 +103,22 @@ public ComponentIdentifier(final ASN1Sequence sequence) throws IllegalArgumentEx
             throw new IllegalArgumentException("Component identifier do not have required values.");
         }
 
+        ASN1Sequence componentIdSeq;
+        int tag = 0;
+
+        if (sequence.getObjectAt(tag) instanceof ASN1Sequence) {
+            componentIdSeq = ASN1Sequence.getInstance(sequence.getObjectAt(tag++));
+            componentClass = DEROctetString.getInstance(componentIdSeq
+                    .getObjectAt(COMPONENT_IDENTIFIER))
+                    .toString();
+        } else if (sequence.getObjectAt(tag) instanceof DEROctetString) {
+            componentClass = sequence.getObjectAt(tag).toString();
+            tag++;
```

Comment:
Couldn't `getObjectAt(tag)` use `tag++`? Or is this for clarity since it's in the middle of the inline function calling.

Reply:
No idea why I did that, the previous line does it in getObjectAt.